src/main/java/api/kirana/register/transaction/service/TransactionServiceImpl.java

```diff
@@ -1,125 +1 @@
-package api.kirana.register.transaction.service;
 
-import api.kirana.register.transaction.models.Transaction;
-import api.kirana.register.transaction.models.TransactionDTO;
-import api.kirana.register.transaction.repo.TransactionDAO;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.cache.annotation.CacheEvict;
-import org.springframework.cache.annotation.CachePut;
-import org.springframework.cache.annotation.Cacheable;
-import org.springframework.stereotype.Service;
-
-import java.time.LocalDate;
-import java.time.LocalTime;
-import java.util.List;
-import java.util.Optional;
-@Service
-public class TransactionServiceImpl implements TransactionService {
-
-    private final TransactionDAO transactionDAO;
-
-    @Autowired
-    public TransactionServiceImpl(TransactionDAO transactionDAO) {
-        this.transactionDAO = transactionDAO;
-    }
-
-    /**
-     * lists all transactions along with their specific details
-     * @return
-     */
-    @Cacheable(value = "Transaction")
```

Comment:
Either use RedisTemplate from caching or use Our frameworks annotation @ReadThroughCache, so that we can control the redis key name